src/debugtopus.ts

```diff
@@ -42,13 +63,14 @@ export const prepareTestRun = async ({
 }> => {
   const code = await getPlaywrightCode(testId, token, url, octomindUrl);
 
-  const dirname = __dirname;
-  let tempDir = path.join(dirname, "..", "..", "..", "..", "temp");
-  if (process.env.NODE_ENV === "test") {
-    tempDir = path.join(dirname, "..", "temp");
+  const nodeModule = require.main;
+  if (!nodeModule) {
+    throw new Error("package was not installed as valid nodeJS module");
   }
-
+  const appDir = dirname(nodeModule.filename);
+  const tempDir = getTempDirOnPackageRootLevel(appDir);
```

Comment:
@Germandrummer92 is this okay method name?

Reply:
i like, yes!